server/api/product/searchRelatedProducts.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import { getGQLClient } from "../../util"
```

Comment:
Change this api to "getProducts", the query you have here is not exclusively for "searchRelatedProducts"